packages/client/src/SubmissionController.test.ts

```diff
@@ -20,14 +20,16 @@ import {
   REGISTER_BIRTH_APPLICATION,
   REJECT_BIRTH_APPLICATION,
   COLLECT_BIRTH_CERTIFICATE,
-  ARCHIVE_BIRTH_DECLARATION
+  ARCHIVE_BIRTH_DECLARATION,
+  REINSTATE_BIRTH_APPLICATION
```

Comment:
We need to refactor `REINSTATE_BIRTH_APPLICATION` to `REINSTATE_BIRTH_DECLARATION`

Reply:
Changed